api/db/changelog/db.changelog-124-add-demographic-gender-identity.xml

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+    xmlns="http://www.liquibase.org/xml/ns/dbchangelog/1.9"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog/1.9
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-1.9.xsd">
+  <changeSet author="nsaxena" id="changelog-124-add-demographic-gender-identity">
+    <createTable tableName="demographic_survey_gender">
+  	  <column name="demographic_survey_id" type="bigint">
+  	 		 <constraints nullable="false"/>
+  	  </column>
+  	  <column name="gender" type="tinyint">
```

Comment:
In the UI it looked like Gender was a checkbox. Should it be a radio button if they're mutually exclusive?

Reply:
Use can select multiple option for GENDER